packages/beacon-node/src/network/peers/peerManager.ts

```diff
@@ -449,12 +460,21 @@ export class PeerManager {
       }
     }
 
+    const status = this.statusCache.get();
+    const starved =
```

Comment:
can we track this in a gauge/counter metric? it's nice to know if the node is "starved" or not when we want to debug a stalled sync